test/__snapshots__/table.test.ts.snap

```diff
@@ -608,12 +608,12 @@ $util.qr($v5.put('consistentRead', $v1.get('consistentRead')))
 $util.qr($v5.put('select', $v1.get('select')))
 #end
 $util.toJson($v5)",
-  "#set( $context.stash.return__flag = true )
-#set( $context.stash.return__val = $context.result.items )
+  "#set( $context.stash.0_tmp = $context.result )
```

Comment:
Is this a regression?

Reply:
Good question, I am going to say "no" because it follows the rules.

Previously 

```ts
return table.query().items
```

now

```ts
var 0_temp = table.query()
// template split
return 0_temp.items;
```

Because we say the only integration calls we keep inline are those directly on the statements. 

This strategy alights with arbitrary complexity before or after the integrations

```ts
return table.query.items[0].S;
```